google-cloud-web_risk-v1/synth.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "webrisk", "v1",
+    extra_proto_files=["google/cloud/common_resources.proto"],
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-web_risk-v1",
```

Comment:
Given that a file path in the current client has this mixture of snake-case and unseparated naming:

`google-cloud-webrisk/lib/google/cloud/webrisk/v1beta1/web_risk_service_v1_beta1_client.rb`

Can you confirm that the casing in `web_risk` and `WebRisk` in this client is correct?

Reply:
Yes. We are intentionally re-casing the naming to standardize on `web_risk` and `WebRisk`. In fact, this migration will be unusual in that the "migrated" client will be a new library `google-cloud-web_risk`, and we will deprecate the old library `google-cloud-webrisk` entirely.